filter1.js

```diff
@@ -24,3 +24,5 @@ var isCool = function(person) {
 }
 
 // Your code goes here
+const newPeople = people.filter(isCool);
+console.log(newPeople)
```

Comment:
nice shortcut here! the 'long way' of writing this out would be:

`const newPeople = people.filter(person => isCool(person))`

You recognized that `filter` takes each of the people in the `people` array and passes them through the callback automatically. 💯